react/ActionMenu/Readme.md

```diff
@@ -19,7 +20,14 @@ const hideMenu = () => setState({ menuDisplayed: false });
       onClose={hideMenu}>
       <ActionMenuItem left={<Icon icon='file' />}>Item 1</ActionMenuItem>
       <ActionMenuItem left={<Icon icon='right' />}>Item 2</ActionMenuItem>
-      <ActionMenuItem left={<Icon icon='file' />}>Item 3</ActionMenuItem>
+      <ActionMenuItem left={<Icon icon='file' />}>
+        <div>
+          Item 3
+          <Caption className="u-maw-5 u-mt-half">
```

Comment:
I don't have a problem with letting the user set a max-width, most of the time there will already be one anyway and if not it's dependent on the situation.

I'm more annoyed by the margin, but again, I figure the margin might not be the same for a description and for other components that are not specified yet… so I thought I'd leave it open as well.

Reply:
I don't understand why we have to do that ?

Reply:
I thought in most cases there would be a parent with a max-width but I was wrong. This makes it worth adding a component for the description imo, I'll add it.

Reply:
I think the max-width should be directly on the ActionMenu.